test/v3/services/CreatePeriodSummaryServiceSpec.scala

```diff
@@ -33,52 +35,75 @@ class CreatePeriodSummaryServiceSpec extends ServiceSpec {
   val businessId: String             = "XAIS12345678910"
   implicit val correlationId: String = "X-123"
 
-  private val requestBody: CreatePeriodSummaryBody =
-    CreatePeriodSummaryBody(
-      PeriodDates("2019-08-24", "2019-08-24"),
-      None,
-      None,
-      None
-    )
+  val periodIncomeWithCl290Enabled: PeriodIncome = PeriodIncome(turnover = Some(2000.00), None, taxTakenOffTradingIncome = Some(2000.00))
+
+  val periodIncomeWithCl290Disabled: PeriodIncome = PeriodIncome(turnover = Some(2000.00), None, taxTakenOffTradingIncome = None)
+
+  private val requestDataWithCl290Enabled = CreatePeriodSummaryRequest(
+    nino = Nino(nino),
+    businessId = BusinessId(businessId),
+    body = CreatePeriodSummaryBody(PeriodDates("2019-08-24", "2019-08-24"), Some(periodIncomeWithCl290Enabled), None, None)
+  )
 
-  private val requestData = CreatePeriodSummaryRequest(
+  private val requestDataWithCl290Disabled = CreatePeriodSummaryRequest(
     nino = Nino(nino),
     businessId = BusinessId(businessId),
-    body = requestBody
+    body = CreatePeriodSummaryBody(PeriodDates("2019-08-24", "2019-08-24"), Some(periodIncomeWithCl290Disabled), None, None)
```

Comment:
Can we pull the body out - this makes it a bit tidied and clearer what changes between the scenarios

```
private val parsedRequestBody = CreatePeriodSummaryBody(PeriodDates("2019-08-24", "2019-08-24"), None, None, None)

  private val requestDataWithCl290Enabled = CreatePeriodSummaryRequest(nino, businessId, parsedRequestBody.copy(income = Some(periodIncomeWithCl290Enabled), None, None)

  private val requestDataWithCl290Disabled = CreatePeriodSummaryRequest(nino, businessId, parsedRequestBody.copy(income = Some(periodIncomeWithCl290Disabled), None, None)

```